en/config/airspeed.md

```diff
@@ -1,6 +1,6 @@
 # AirSpeed Calibration
 
-For airspeed calibration you will need to keep your airspeed sensor out of any wind and then blow across the sensor.
+The airspeed calibration needs to read a stable baseline with 0 airspeed to determine an offset. Cup your hands over the pitot to make sure it's not reading any wind (if calibrating the sensor inside this is not needed) and then blow into the pitot to signal completion of the calibration.
```

Comment:
Hi @hamishwillee, Suggestion: 
inside --> indoor
blow into the pitot --> blow air by mouth into the tip of the pitot tube

Reply:
Cheers. I made these changes - though to be very pedantic indoors and inside are used pretty much interchangeably, and we don't really care how you blow into the tube :-)